packages/extension-core/src/types/index.ts

```diff
@@ -9,14 +9,15 @@ import { EncryptMessages } from "../domains/encrypt/types"
 import { EthMessages } from "../domains/ethereum/types"
 import { MetadataMessages } from "../domains/metadata/types"
 import { MnemonicMessages } from "../domains/mnemonics/types"
+import { NftsMessages } from "../domains/nfts"
 import { SigningMessages } from "../domains/signing/types"
 import { AuthorisedSiteMessages } from "../domains/sitesAuthorised/types"
 import { SubstrateMessages } from "../domains/substrate/types"
 import { TalismanMessages } from "../domains/talisman/types"
 import { TokenRatesMessages } from "../domains/tokenRates/types"
 import { TokenMessages } from "../domains/tokens/types"
 import { AssetTransferMessages } from "../domains/transfers/types"
-import type { IdOnlyValues, NoUndefinedValues, NullKeys, RequestIdOnly } from "./base"
+import type { IdOnlyValues, NoUndefinedValues, NullKeys, Port, RequestIdOnly } from "./base"
```

Comment:
No need to import `Port` anymore, you can just use `chrome.runtime.Port` which is global